### ISSUE TITLE
Non Hinting Regression Test

### ISSUE THREAD

**Issue by davelab6** [latofonts/lato-source#3]
@m4rc1e please could you do a regression test without hinting, to check the outlines are good to go to the hinting phase? :)

**Comment by davelab6** [comment]
@twardoch mentioned the spacing is a little tighter in this v3 rather than v1. 

This also has a family name `Lato VF` inside the TTFs, that will be updated as part of the final range of changes

**Comment by davelab6** [comment]
The statics are not instantiated from the VF; there is `mkmk` stuff in the statics that is not in the VFs. Adam did this because fontttols can't do it; the GPOS stuff are different in different masters. So FontLab disables mark feature generation for VFs, as fontTools fails to handle about 50% of cases there. This makes the filesize large.

**Comment by twardoch** [comment]
Yes, the static fonts are built with FontLab, so their outlines aren't identical. Overlaps are removed. 

I intend to play a bit with customized cubic-to-quadratic conversion for Lato so I can see how much I can reduce the size without sacrificing quality. Since the Thins are pretty thin, I wouldn't be too hopeful.

Thin fonts are more sensitive to small deviations in segment shape resulting from reducing the TT offcurve points. I actually think that this is something that could be developed into a tool (for fontmake etc), I mean some adaptive cu2qu conversion.

I'll also compare FontLab’s and cu2qu’s conversion to determine the optimal route. 

Right now, the mark/mkmk code resulting from Lato is OK for statics but varLib fails to merge them. Also, FontLab may just do too much. I'll take a look at the sizes of various parts of the VF. For example, FL7 now builds mark and also mkmk for all kinds of glyphs that have anchors. But perhaps we can find some way to limit it so only some glyphs (both bases and marks) get mark/mkmk. Right now mark attachment is at "it works" level but it's not always sensible — but may contribute to the size significantly.
